templates/deployment.yml

```diff
@@ -37,3 +11,253 @@ update:
+      common:
+        tls_cacert: ((common.tls_cacert))
+        tls_client_cert: ((common.tls_client_cert))
+        tls_client_key: ((common.tls_client_key))
+      internal:
+        ip: default
+        log_event: true
+        port: default
+        ssl:
+          cert: ((ssl.cert))
+          key: ((ssl.key))
+      external:
+        api_requires_admin_scope: false
+        cookie_secret: default
+        host: ((broker_host))
+        log_event: true
+        port: default
+        trust_proxy: 2
+      mongodb:
+        agent:
+          auth:
+            password: default
+            username: default
+          provider:
+            container: default
+          supported_features:
+          - state
+          - lifecycle
+          - credentials
+          - backup
+          - restore
+          version: "1"
+        backup:
+          backup_timeout_time: default
+          schedule_interval: default
+          status_check_every: 120000
+        deployment_name: service-fabrik-mongodb
+        provision:
+          network_index: 1
+        record_max_fetch_count: 100
+        retry_connect:
+          max_attempt: 8
+          min_delay: 120000
+      cf:
+        password: default
+        url: ((cf_url))
+        username: default
+      syslog:
+        host: default
+        port: default
+      docker:
+        allocate_docker_host_ports: true
+        url: default
+      quota:
+        whitelist: null
+        oauthDomain: null
+        serviceDomain: null
+        username: null
+        password: null
+      directors:
+      - cpi: warden_cpi
+        name: bootstrap-bosh
+        password: default
+        primary: false
+        skip_ssl_validation: true
+        support_create: false
+        url: default
+        username: default
+      - default_task_poll_interval: 10000
+        infrastructure:
+          azs:
+          - cloud_properties:
+              name: random
+            name: z1
+          - cloud_properties:
+              name: random
+            name: z2
+          - cloud_properties:
+              name: random
+            name: z3
+          compilation:
+            cloud_properties:
+              name: random
+            network: compilation
+            reuse_compilation_vms: true
+            workers: 4
+          networks:
+          - name: service-fabrik-bosh-services
+            subnets: default
+            type: manual
+          - name: compilation
+            type: dynamic
+          segmentation:
+            capacity: -1
+            network_name: service-fabrik-bosh-services
+            offset: 1
+            size: 8
+          stemcell:
+            alias: default
+            os: ubuntu-trusty
+            version: latest
+        lock_deployment_max_duration: 86400
+        name: bosh
+        password: default
+        primary: true
+        skip_ssl_validation: true
+        support_create: true
+        url: default
+        username: default
+      services:
```

Comment:
This would still be needed? I thought we are doing it through render script??

Reply:
this is just a dummy placeholder. I will set the value as `default`. the idea is that this value will be replaced by whatever is provided from the ops file.